tests/generic/bug01425.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+Test for PHP-1425: explain() does not raise appropriate exception for $err conditions
+--SKIPIF--
+<?php require_once "tests/utils/standalone.inc"; ?>
+--FILE--
+<?php
+require_once "tests/utils/server.inc";
+
+$host = MongoShellServer::getStandaloneInfo();
+$m = new MongoClient($host);
+
+$m->admin->command( array( 'setParameter' => 1, 'notablescan' => 1 ) );
+
+$c = $m->selectCollection(dbname(), collname(__FILE__));
+
+$c->drop();
+$c->insert( array( 'foo' => 'bar' ) );
+try
+{
+	$c->find( array( 'foo' => 'bar' ) )->explain();
+}
+catch ( MongoCursorException $e )
+{
+	echo $e->getCode(), "\n";
+
+	$m->admin->command( array( 'setParameter' => 1, 'notablescan' => 0 ) );
+}
+
+$m->admin->command( array( 'setParameter' => 1, 'notablescan' => 0 ) );
+?>
```

Comment:
It doesn't have to be both in the catch() and outside it.
Also, on failure this test won't print anything - thats abit confusing. how about ending with echo ok or ===DONE=== ? :]